README.yaml

```diff
@@ -92,6 +92,11 @@ introduction: |-
 
   For example, maybe you have your own existing CLI tools (e.g. using `terragrunt`). In this case, you may want to start by developing your own CLI.
 
+  ## Install
+
```

Comment:
### Install with homebrew

```
  brew install atmos
  ```

Maybe also add instructions on how to install a specific version (if it's easy to explain and to do)?

Reply:
I added golang instructions too